vowpalwabbit/parse_args.cc

```diff
@@ -1201,6 +1201,8 @@ vw& parse_args(int argc, char *argv[], trace_message_t trace_listener, void* tra
     ("initial_regressor,i", po::value< vector<string> >(), "Initial regressor(s)")
     ("initial_weight", po::value<float>(&(all.initial_weight)), "Set all weights to an initial value of arg.")
     ("random_weights", po::value<bool>(&(all.random_weights)), "make initial weights random")
+    ("normal_weights", po::value<bool>(&(all.normal_weights)), "make initial weights normal")
+    ("truncated_normal_weights", po::value<bool>(&(all.tnormal_weights)), "make initial weights truncated normal")
```

Comment:
correct for now it does default mean=0, sd=1.
Implemented the way it is defined in tensorflow docs for default.
(https://www.tensorflow.org/api_docs/python/tf/truncated_normal)
will add user control of "mean" and "sd" values in the next iteration.